lxd/instance/drivers/driver_qemu.go

```diff
@@ -1371,6 +1371,20 @@ func (d *qemu) Start(stateful bool) error {
 		return err
 	}
 
+	// Get the list of PIDs from the VM.
+	pids, err := monitor.GetCPUs()
+	if err != nil {
+		op.Done(err)
+		return err
+	}
+
+	err = d.setCoreSched(pids)
+	if err != nil {
+		err = fmt.Errorf("Failed to allocate new core scheduling domain for vCPUs threads")
```

Comment:
Please can you wrap the error from `setCoreShed` using `fmt.Errorf("Failed to allocate new core scheduling domain for vCPUs threads: %w", err)`

Reply:
Yep, will do.